README.md

```diff
@@ -511,9 +511,18 @@ This library currently does not support being the [target of a push notification
 
 - ["Unable to parse request body" error when publishing large messages from old versions of Internet Explorer](https://support.ably.com/solution/articles/3000062360-ably-js-unable-to-parse-request-body-error-when-publishing-large-messages-from-old-browsers).
 
-#### Manifest v3 Chrome Extensions
-Chrome extensions built with Manifest v3 require service workers instead of background pages.
-This is supported in Ably via the [Web Worker build](#supported-platforms), however [workarounds](docs/chrome-mv3.md) are required to ensure Chrome does not mark the service worker as inactive.
+#### Chrome Extensions
+Ably supports Chrome extensions out-of-the-box since v2. However, extension service workers get improved support for
```

Comment:
It's ambiguous whether you're talking about ably-js v2 or manifest v2, moreover we should mention that it works because you can load ably-js in background scripts.
```suggestion
ably-js works out-of-the-box in background scripts for Chrome extensions using manifest v2. However, extension service workers get improved support for
```

Moreover,